package.json

```diff
@@ -22,10 +24,23 @@
   },
   "homepage": "https://github.com/Visual-Testing-for-Android-Apps/backend#readme",
   "devDependencies": {
-    "npm-run-all": "^4.1.5"
+    "husky": "^7.0.1",
+    "lint-staged": "^11.1.1",
+    "npm-run-all": "^4.1.5",
+    "prettier": "^2.3.2",
+    "standard": "^16.0.3",
+    "ts-standard": "^10.0.0"
   },
   "dependencies": {
     "axios": "^0.21.1",
     "express": "^4.17.1"
+  },
+  "lint-staged": {
+    "**/*": "prettier --write --ignore-unknown",
+    "**/*.js": "standard --fix",
+    "**/*.ts": "ts-standard --fix"
+  },
+  "prettier": {
+    "printWidth": 100
```

Comment:
Is it possible to use “tabWidth”: 4

Reply:
JSON files look horrible with a `"tabWidth”: 4`, so the default behaviour is fine.

Reply:
I disagree, and find a tab width of 2 completely unreadable, but I was waiting for other peoples opinions.
However, if we use "useTabs: true" then both of use could be happy (this only affects indentation, spaces are still used for alignment).

Reply:
Ok, I'm convinced, made the changes required.